### PR TITLE
breaking: updating classnames for better BEM syntax

### PR DESCRIPTION
**Pull request by dbanksdesign**
#### Description of changes

Cleaning up some classnames and CSS for primitives that didn't  do BEM properly

#### Checklist

- [x] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [x] PR description included
- [x] Relevant documentation is changed or added (and PR referenced)
- [x] `yarn test` passes and tests are updated/added
- [x] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by reesscot — DISMISSED** [review]

File: packages/react/src/primitives/Divider/Divider.tsx [hunk 1]
```diff
@@ -19,7 +19,6 @@ const DividerPrimitive: Primitive<DividerProps, 'hr'> = (
     ComponentClassName.Divider,
     classNameModifier(ComponentClassName.Divider, orientation),
     classNameModifier(ComponentClassName.Divider, size),
-    label ? ComponentClassName.DividerLabel : null,
```
Comment (reesscot) [inline packages/react/src/primitives/Divider/Divider.tsx:22]: what does this have to do with bem?

Comment (dbanksdesign) [reply packages/react/src/primitives/Divider/Divider.tsx:22]: Nothing, just unnecessary. Was able to remove some CSS

Comment (ioanabrooks) [reply packages/react/src/primitives/Divider/Divider.tsx:22]: Are we sure it's not necessary? Think it's used in the federated signin component.

Comment (dbanksdesign) [reply packages/react/src/primitives/Divider/Divider.tsx:22]: I updated the CSS to remove that classname: https://github.com/aws-amplify/amplify-ui/pull/4474/files#diff-30ce68f480e6f1f48bb3548f6d48300e8bcb87b2c39e83774f81564c0d67ced0R16

**Review by reesscot — APPROVED** [review]

**Review by hbuchel — APPROVED** [review]
LGTM!
